### DISCUSSION TITLE
Wireframes

### DISCUSSION THREAD

**Discussion by AdrienDSlone** [AdrienDSlone/slowTripPlanner#41] [3. UI/UX]
Bonsoir à tous,

je vous partage les wireframes adaptées à la maquette que je vous avais partagée :

## Avec la maquette
![20220627_wireframe_above_mockup](https://user-images.githubusercontent.com/50139099/176249335-a4fd742b-cc66-4d61-88c9-f5f0616564a0.png)

## Sans la maquette 
![20220627_wireframe](https://user-images.githubusercontent.com/50139099/176249343-a1e3d939-d6cc-45a2-8f01-336f96493ee0.png)

**Comment by meiry11** [comment]
c'est la page home?
ou se sont des composants qui nous serviront de page après

**Reply by AdrienDSlone** [reply]
La page d'un parcours et les composants et ou le html qui la constituent
